examples/Makefile

```diff
@@ -0,0 +1,13 @@
+all: cvc4_qf_ufbv btor_qf_ufbv
+
+# Note: assumes smt-switch has been installed in a directory called
+# example-install in this directory
```

Comment:
maybe mention here that the script `build.sh` does just that?

Reply:
Good idea -- addressed with c2b862a13b2101020b66ded1942118f9002f718f